x/evm/core/core/utils.go

```diff
@@ -0,0 +1,81 @@
+package core
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/evmos/evmos/v18/x/evm/core/vm"
+)
+
+// NewEVMBlockContext creates a new context for use in the EVM.
+func NewEVMBlockContext(header *types.Header, chain core.ChainContext, author *common.Address) vm.BlockContext {
```

Comment:
Where is this used?

Reply:
Nice catch Rama, addressed here:

https://github.com/evmos/evmos/pull/2661